examples/tof-viewer/README.md

```diff
@@ -66,16 +66,14 @@ Refer to Figure 5.
 
 **Figure 5**: *Recording Options*
 
-Once the above options are set, then press the "Start Recording" button. The Viewer GUI will prompt a "Save As" window to pre-save the file name. It is advisable to give a unique name to the new recording file. Then, choose file format, which can be either *.raw (binary) format or *.fsf format, as depicted in Figure 6.
+Once the above options are set, then press the "Start Recording" button. The Viewer GUI will prompt a "Save As" window to pre-save the file name. It is advisable to give a unique name to the new recording file. Then, choose file format, which can be *.raw (binary) format, as depicted in Figure 6.
```

Comment:
You need to replace the Figure 6 with another screen capture that shows only 'raw' option. It believe that the same applies to figure 8.